### ISSUE TITLE
Saving of frames

### ISSUE THREAD

**Issue by thisisjasleen** [KaiyangZhou/pytorch-vsumm-reinforce#24]
Can anyone please tell me how are the frames being saved to .jpg images?

**Comment by SinDongHwan** [comment]
you should download dataset and convert frames from videos.
and when run summary2video.py , you should edit read frame in summary2video.py.

**Comment by bhardwaj007** [comment]
By frames, i mean those frames which are to be selected in the summary, how can we save them as jpg images?

**Comment by thisisjasleen** [comment]
@SinDongHwan
By frames i mean, those frames which are to be added in the summary, how are we suppsed to save them as jpg images?

**Comment by SinDongHwan** [comment]
Did you download original dataset(SumMe)?
In original dataset, shell script exists about video to frames.

**Comment by thisisjasleen** [comment]
Yes, i did download the SumMe dataset. But there is no available shell script ?

**Comment by SinDongHwan** [comment]
you can find shell script in video directory.

**Comment by thisisjasleen** [comment]
Can you send me the shell script?

**Comment by SinDongHwan** [comment]
i can’t , because i can’t use internet. 
you can find it.

**Comment by thisisjasleen** [comment]
for _ in range(args.num_episode)
This line in main.py.. is there supposed to be _ in the for loop?

**Comment by thisisjasleen** [comment]
@SinDongHwan 
Can you please tell me which CUDA version to use for running these codes?
I have cuda 10.0 installed and I'm afraid pytorch v0.4.0 is not compatible with it.

**Comment by SinDongHwan** [comment]
@thisisjasleen 
maybe... CUDA 9.0? 9.2?
i don't try cuda10

**Comment by chenchch94** [comment]
@SinDongHwan Hi，Can you provide the datasets?please，it is 404 Not found when I download it.Thank you very much.

**Comment by 18pxrsun** [comment]
> Did you download original dataset(SumMe)?
> In original dataset, shell script exists about video to frames.

Could you give me the link to download the original dataset?
